.github/workflows/linter.yml

```diff
@@ -0,0 +1,16 @@
+name: Linter
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
as it's just a lint task, should be fine to run for other branches as well (if we ever start using multiple branches)